exercises/4-an-enslaved-hope/README.md

```diff
@@ -45,171 +45,113 @@ _____
-git checkout master
-```
-```bash
-git merge develop
+git commit -m "ADD - Arachni scanning image"
 ```
 ```bash
-# this is to delete the branch from the remote
-git push origin :feature/jenkinsfile
+git push
 ```
+
+3. Run the Ansible playbook filtering with tag `arachni` so only the arachni build pods are run.
 ```bash
-git push -u origin --all
-```
+ansible-playbook apply.yml -e target=tools \
+     -i inventory/ \
+     -e "filter_tags=arachni"
+``` -->
 
-18. On Jenkins; create a new `Multibranch Pipeline` job called `todolist-fe`.
+2. Head to <CLUSTER_URL> on OpenShift and move to your ci-cd project `Builds > Images`. You should see the `jenkins-slave-arachni` image.
+![builds-zap-arachni](../images/exercise4/builds-zap-arachni.png)
 
-19. Add the `todolist-fe` git repository and set the credentials for git accordingly.
+3. Just like you did with the `jenkins-slave-npm`, configure the `jenkins-slave-arachni` pod template to bypass SSL certificate checks in the Jenkins global configuration settings. Log in to Jenkins and navigate to `Manage Jenkins` > `Configure System` page.
 
-20. Set the trigger to scan every minute as done previously. Save the configuration and we should see the collection of Jobs as shown below.
-![todolist-fe-multi](../images/exercise4/todolist-fe-multi.png)
+4. Locate the `Kubernetes Pod Template` section by scrolling to the bottom of the page and click on `Add Pod Template` to add a new pod template for the Arachni scanner slave.
+![add-kube-pod-template](../images/exercise4/add-kube-pod-template.png)
 
-> NOTE: If the `feature/jenkinsfile` job is running, you can stop it to speed up the job execution for the `master` and `develop` branches.
+5. For the new pod template, enter `jenkins-slave-arachni` in the `Name` and `Labels` fields. 
 
-21. Run the jobs and validate the app is working as expected in the `test` environment!
-![todolist-test](../images/exercise4/todolist-test.png)
+6. In the `Containers` section for the pod template, add a new container template with the following details:
+    * Enter `jnlp` in the `Name` field
+    * Enter `docker-registry.default.svc:5000/<YOUR_NAME>-ci-cd/jenkins-slave-arachni` in the `Docker image` field
+    * Enter `/tmp` in the `Working directory` field
+    * Enter `${computer.jnlpmac} ${computer.name}` in the `Arguments to pass to the command` field
+
+7. Add a new environment variable for the container template called `GIT_SSL_NO_VERIFY` and set its value to `true`. Your final `jenkins-slave-arachni` kubernetes pod template should look like the following:
+![add-kube-pod-template](../images/exercise4/new-arachni-container-template.png)
+
+8. Click `Save` at the bottom of the page to save your global Jenkins settings.
 
 ### Part 2 - OCP Pipeline
 > _This exercise adds a new BuildConfig to our cluster for the todolist-apps to run their pipelines in OpenShift using the OpenShift Jenkins Sync Plugin. We will use the OpenShift Applier to create the content in the cluster_
```

Comment:
do a `ctrl+f` for `todolist-fe` (below)

Reply:
As per discussion - this needs to happen errywhere